server/WebAPI/Services/ScheduledPlaceToEatService.cs

```diff
@@ -0,0 +1,254 @@
+      }
+      var fileNames = new List<string>();
+      var attachmentFileEatings = _attachmentFileEatingRepository.GetAttachmentFileEatingByScheduledPlaceId(scheduledPlaceToEat.Id);
+      if(attachmentFileEatings != null)
+      {
+        foreach(var file in attachmentFileEatings)
+        {
+          fileNames.Add(file.Path);
+        }
+      }
+      return new OutputScheduledPlaceToEatDTO
+      {
+        Id = scheduledPlaceToEat.Id,
+        DateTime = scheduledPlaceToEat.DateTime,
+        FileNames = fileNames,
+        GooglePlaceId = scheduledPlaceToEat.GooglePlaceId,
+        Lat = scheduledPlaceToEat.Lat,
+        Lng = scheduledPlaceToEat.Lng,
+        Link = scheduledPlaceToEat.Link,
+        NamePlace = scheduledPlaceToEat.NamePlace,
+        Notes = scheduledPlaceToEat.Notes
+      };
+    }
+
+    public ScheduledPlaceToEat ConvertInputScheduledPlaceToEatForCreateDTOToScheduletPlaceToEat(InputScheduledPlaceToEatForCreateDTO inputScheduledPlaceToEatDTO)
+    {
+      if(inputScheduledPlaceToEatDTO is null)
+      {
+        throw new ArgumentNullException(nameof(inputScheduledPlaceToEatDTO));
+      }
+      return new ScheduledPlaceToEat
+      {
+        DateTime = inputScheduledPlaceToEatDTO.DateTime,
+        GooglePlaceId = inputScheduledPlaceToEatDTO.GooglePlaceId,
+        Lat = inputScheduledPlaceToEatDTO.Lat,
+        Lng = inputScheduledPlaceToEatDTO.Lng,
+        Link = inputScheduledPlaceToEatDTO.Link,
+        NamePlace = inputScheduledPlaceToEatDTO.NamePlace,
+        Notes = inputScheduledPlaceToEatDTO.Notes,
+        UserId = inputScheduledPlaceToEatDTO.UserId
+      };
+    }
+
+    public ScheduledPlaceToEat ConvertInputScheduledPlaceToEatForUpdateOrDeleteDTOToScheduletPlaceToEat(InputScheduledPlaceToEatForUpdateOrDeleteDTO inputScheduledPlaceToEatDTO)
+    {
+      if(inputScheduledPlaceToEatDTO is null)
+      {
+        throw new ArgumentNullException(nameof(inputScheduledPlaceToEatDTO));
+      }
+      return new ScheduledPlaceToEat
+      {
+        Id = inputScheduledPlaceToEatDTO.Id,
+        DateTime = inputScheduledPlaceToEatDTO.DateTime,
+        GooglePlaceId = inputScheduledPlaceToEatDTO.GooglePlaceId,
+        Lat = inputScheduledPlaceToEatDTO.Lat,
+        Lng = inputScheduledPlaceToEatDTO.Lng,
+        Link = inputScheduledPlaceToEatDTO.Link,
+        NamePlace = inputScheduledPlaceToEatDTO.NamePlace,
+        Notes = inputScheduledPlaceToEatDTO.Notes,
+        UserId = inputScheduledPlaceToEatDTO.UserId
+      };
+    }
+
+    private bool Valid(InputScheduledPlaceToEatForUpdateOrDeleteDTO scheduledPlaceToEatDTO)
+    {
+      var results = new List<ValidationResult>();
+      var context = new ValidationContext(scheduledPlaceToEatDTO);
+      try
+      {
+        if(!Validator.TryValidateObject(scheduledPlaceToEatDTO, context, results, true))
+        {
+          foreach(var error in results)
+          {
+            Console.WriteLine(error.ErrorMessage);
```

Comment:
why do we need to do console.writeline here?

Reply:
ok